lambda/solana-memo-lambda-invoke/scripts/logMemo.js

```diff
@@ -0,0 +1,36 @@
+import env from "dotenv";
+import { invokeSolana } from "../../lambdaUtils.mjs";
+import {
+  waitUntilTransactionMined,
+  checkAPIKey,
+} from "../../../helpers/utils.mjs";
+
+import logMemoSerializedMessage from "../backend/utils.cjs";
+import getDeveloperWallet from "../../../helpers/utils.mjs";
+
+async function main() {
```

Comment:
doesn't seem to be working
<img width="1192" alt="image" src="https://github.com/PassbirdCo/MetaKeepCodeSamples/assets/100057389/42b48f8a-142c-4e9e-819c-d285da7bd280">

Reply:
It's the same issue that I fixed earlier this week. `import` or `require` only works in files below the current directory.